lib/postgres_v3_experimental.dart

```diff
@@ -128,11 +128,26 @@ abstract class PgStatement {
   Future<void> dispose();
 }
 
-class PgTypedParameter {
+/*final*/ class PgTypedParameter {
```

Comment:
nit: remove comment or make actually final?

Reply:
I wanted to leave it as a note for later since the package hasn't been migrated to Dart 3 yet, but that turned out to be very simple so I've applied `final` modifiers to the v3 API where appropriate.